tests/kuttl/tests/cinder_scale/01-assert.yaml

```diff
@@ -0,0 +1 @@
+/home/anbanerj/Documents/next-gen/cinder-operator/tests/kuttl/common/assert_sample_deployment.yaml
```

Comment:
This should be a relative path, i.e. `../../common/assert_sample_deployment.yaml

Reply:
ack, done,  thanks. Somehow it kept on taking the whole path from my local when I tried to create a symlink. I amended the multiple commits and merged the changes into one now.